app/controllers/orders_controller.rb

```diff
@@ -20,6 +19,7 @@ def index
 
   def show
     @order = Order.find(params[:id])
+    @status = @order.status.name
```

Comment:
Question: is calling ".name" in the controller necessary when we pass the status into the view?